nasa-gibs/worldview
# WV-3285 Pre-Caching Nearby Non-Subdaily Layers Fix

## Description
This change fixes the caching behavior for non subdaily layers when changing days using the arrows next to the timeline.

## How To Test
1. `git checkout wv-3285-image-pre-fetch`
2. `npm ci`
3. `npm run watch`
4. Open [this link](http://localhost:3000/?t=2024-07-25-T14%3A49%3A55Z)
5. Let the imagery fully load (a couple of seconds)
6. Click on the left and right arrows next to the timeline to change the date to nearby days, and verify that the imagery is pre-cached for those other days
